<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="widgetplotX" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>761</width>
      <height>111</height>
     </rect>
    </property>
    <widget class="QPushButton" name="xAxisBtn">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>X Axis</string>
     </property>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="widgetplotY" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>761</width>
      <height>111</height>
     </rect>
    </property>
    <widget class="QPushButton" name="yAxisBtn">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Y Axis</string>
     </property>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="widgetplotZ" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>761</width>
      <height>111</height>
     </rect>
    </property>
    <widget class="QPushButton" name="zAxisBtn">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Z Axis</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="resetBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widgetplotO" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>761</width>
      <height>111</height>
     </rect>
    </property>
    <widget class="QPushButton" name="oAxisBtn">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>O Axis</string>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="rdbtnRawData">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Raw Data</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbtnFiltData">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Filt Data</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbtnFFT">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>FFT</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widgetPlotZoom" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>761</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCutDataBeginWith">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 122, 55);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCutDataBeginWith">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Begin</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCutDataEndWith">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 122, 55);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCutDataEndWith">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>End</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCutData">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cut</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResetInput">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ret</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>927</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionFeatures"/>
    <addaction name="actionReport"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionFeatures">
   <property name="text">
    <string>Features</string>
   </property>
  </action>
  <action name="actionReport">
   <property name="text">
    <string>Report</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
